functionnal_tests/cypress/integration/generic_frontend/operation_tinymce.js

```diff
@@ -125,7 +125,7 @@ describe('TinyMce text editor', function () {
           .then(editor => {
             editor.setContent('Hello')
           })
-        cy.get('[title="More..."]').click()
+        cy.get('[title="More..."]').eq(0).click()
```

Comment:
This isn't very robust because if someone adds an element with the right title before the one you expect in your test, it will break the test.
solution would be to have a more specific selector.
This comment isn't "required change" though, it will depends on the complexity to have such selector.
I let you judge